docroot/config/staging/core.extension.yml

```diff
@@ -13,6 +13,7 @@ module:
   datetime: 0
   datetime_range: 0
   dblog: 0
+  dcc_gtd_multistep: 0
```

Comment:
Please remove the gtd because this module is not necessary related to training days.